### ISSUE TITLE
contributte:cache:generate: Service of type 'Symfony\Component\Console\Application' not found.

### ISSUE THREAD

**Issue by racinmat** [contributte/console-extra#36]
When running `bin/console contributte:cache:generate`, I get following error:
```bash
root@908b9b84ae4c:/var/www/html# bin/console contributte:cache:generate
Compiling Latte templates
All templates successfully compiled.
Compiling DI containers
Compiling container `debug`

In Autowiring.php line 53:
                                                                      
  Service of type 'Symfony\Component\Console\Application' not found.  
                                                                      

contributte:cache:generate [-l|--list] [-g|--generator GENERATOR]
```

This issue is replicable using https://github.com/contributte/webapp-skeleton

after I added 
- row `$configurator->addServices(['configurator' => $configurator]);` to `app/bootstrap.php`

- to `app/config/app/services.neon` section 
```neon
services:
	configurator:
		type: Nette\Configurator
		imported: true
``` 

- to `app/config/ext/contributte.neon`.
```neon
contributte.console.extra:
	advancedCache:
		generators:
			latte: Contributte\Console\Extra\Cache\Generators\LatteTemplatesCacheGenerator(
				@Nette\Application\UI\ITemplateFactory,
				[%appDir%],
				[],
				::realpath(%appDir%/..)
			)
			di: Contributte\Console\Extra\Cache\Generators\DiContainersCacheGenerator(
				[
					debug: [debugMode: true, consoleMode: false],
					production: [debugMode: false, consoleMode: false],
					console: [debugMode: true, consoleMode: true]
				]
			)
``` 

I'm not sure why exactly it's happening, but it's happening me always and I can't figure out any fix. 
Using version 0.6.0 of `contributte/console-extra` and 0.9.1 of `contributte/console-extra`

**Comment by f3l1x** [comment]
Can you please create a PR to https://github.com/contributte/webapp-skeleton? I will test it.

**Comment by racinmat** [comment]
Done https://github.com/contributte/webapp-skeleton/pull/218

**Comment by f3l1x** [comment]
Solved via contributte/console-extra:v0.7.0.

NEON should look like:

```
extensions:
	contributte.console: Contributte\Console\DI\ConsoleExtension(%consoleMode%)
	contributte.console.extra: Contributte\Console\Extra\DI\ConsoleBridgesExtension(%consoleMode%)
```

and

```
extensions:
	nettrine.annotations: Nettrine\Annotations\DI\AnnotationsExtension
	nettrine.cache: Nettrine\Cache\DI\CacheExtension
	nettrine.migrations: Nettrine\Migrations\DI\MigrationsExtension
	nettrine.fixtures: Nettrine\Fixtures\DI\FixturesExtension

	# Dbal
	nettrine.dbal: Nettrine\DBAL\DI\DbalExtension
	nettrine.dbal.console: Nettrine\DBAL\DI\DbalConsoleExtension(%consoleMode%)

	# Orm
	nettrine.orm: Nettrine\ORM\DI\OrmExtension
	nettrine.orm.cache: Nettrine\ORM\DI\OrmCacheExtension
	nettrine.orm.console: Nettrine\ORM\DI\OrmConsoleExtension(%consoleMode%)
	nettrine.orm.annotations: Nettrine\ORM\DI\OrmAnnotationsExtension
```

Focus on `%consoleMode%` in constructors.

Thank you for report.
